godel_plugins/src/widgets/robot_blending_widget.cpp

```diff
@@ -53,11 +69,17 @@ void RobotBlendingWidget::init()
 	robot_scan_config_window_= new RobotScanConfigWidget(robot_scan_parameters_);
 	surface_detect_config_window_ = new SurfaceDetectionConfigWidget(surf_detect_parameters_);
 
+	robot_blend_config_window_= new BlendingPlanConfigWidget(blending_plan_parameters_);
+	scan_plan_config_window_=new ScanPlanConfigWidget(scan_plan_parameters_);
 
 	// setting signals and slots
 	connect(robot_scan_config_window_,SIGNAL(parameters_changed()),this,SLOT(robot_scan_params_changed_handler()));
 	connect(surface_detect_config_window_,SIGNAL(parameters_changed()),this,SLOT(surface_detect_params_changed_handler()));
 	connect(ui_.PushButtonMoreOptions,SIGNAL(clicked()),this,SLOT(scan_options_click_handler()));
+
+	connect(ui_.PushButtonBlendOptions,SIGNAL(clicked()),this,SLOT(blend_options_click_handler()));
+	connect(ui_.pushButtonProfileOptions,SIGNAL(clicked()),this,SLOT(scan_plan_options_click_handler()));
```

Comment:
Perhaps we should decide on a naming convention for these widget elements as lacking such a convention will lead to inconsistent naming practices.  I propose that we use the camel case convention , where the last word is indicative of the widget type, as follows : `ui_.FixEverythingMagicalButton`